job/job.go

```diff
@@ -100,55 +100,39 @@ func (j Job) MarkDeviceActive() {
 //	j.postFailure(reason)
 //}
 
-func (j *Job) setup(d *packet.Discovery) error {
-	mode, netConfig := d.Mode(j.mac), d.NetConfig(j.mac)
-	j.Logger = joblog.With("mac", j.mac, "hardware.id", d.Hardware.ID)
+func (j *Job) setup(dp *packet.Discovery) error {
```

Comment:
It will be nice to have a comments for this function, what will it do? processing dhcp discovery packet? not looks like it .